server/sonar-server/src/main/java/org/sonar/server/computation/step/AbstractComputationSteps.java

```diff
@@ -37,15 +35,14 @@ protected AbstractComputationSteps(ContainerPopulator.Container container) {
 
   @Override
   public Iterable<ComputationStep> instances() {
-    return Iterables.transform(orderedStepClasses(), new Function<Class<? extends ComputationStep>, ComputationStep>() {
-      @Override
-      public ComputationStep apply(@Nonnull Class<? extends ComputationStep> input) {
+    return Iterables.transform(
+      orderedStepClasses(),
+      input -> {
         ComputationStep computationStepType = container.getComponentByType(input);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 7. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)